### PR TITLE
Make linter happy

### PR DESCRIPTION
**Pull request by eljenso**
Fixed some stuff eslint complained about.

### CODE REVIEW

File: vue/src/classes/DeckInspector.js [hunk 1]
```diff
@@ -171,11 +171,18 @@ class DeckInspector {
       let influencePool = this.getInfluencePool();
 
       const offclans = DeckInspector.findSlotsOffClan(conflictDeck, this.clan);
-      for (const slot of offclans) {
```
Comment (Alsciende) [inline vue/src/classes/DeckInspector.js]: sometimes I really hate those opinionated linters

Comment (eljenso) [reply vue/src/classes/DeckInspector.js]: Yeah, I know. Could change the linter config to our liking to save us from headaches in the future.

**Comment by Alsciende** [comment]
I'm really not convinced it's a good idea to lint everything. Why didn't you limit it to `src/`?

**Comment by Alsciende** [comment]
In the end I only made linter happy in /src and /test. I cherry-picked your third commit and fixed a couple things, it's good now. Thanks!

**Pull request merged**